2-copy-of-code/lesson-14/scripts/checkout.js

```diff
@@ -132,6 +132,10 @@ document.querySelectorAll('.js-update-link')
   .forEach((link) => {
     link.addEventListener('click', () => {
       const productId = link.dataset.productId;
-      console.log(productId);
+
+      const container = document.querySelector(
+        `.js-cart-item-container-${productId}`
+      );
+      container.classList.add('is-editing-quantity');
```

Comment:
checkout.js:141 Uncaught DOMException: Failed to execute 'querySelector' on 'Document': '.js-cart-item-container-{productId}' is not a valid selector.
i wrote the same code as above
why my codes go wrong like these?